src/address.rs

```diff
@@ -210,6 +214,34 @@ impl DerivedAddress {
     }
 }
 
+// Locally computes the address for a Cosmos ModuleAccount, which is the first 20 bytes of
+// the sha256 hash of the name of the module.
+// See Module() for more info: https://github.com/cosmos/cosmos-sdk/blob/main/types/address/hash.go
+//
+// Note: some accounts like the Distribution module's "fee_collector" work the same way,
+// despite the fact that "fee_collector" is not a module
+pub fn get_module_account_address(
+    module_name: &str,
+    prefix: Option<&str>,
+) -> Result<Address, AddressError> {
+    let prefix = prefix.unwrap_or(DEFAULT_PREFIX);
+
+    // create a Sha256 object
+    let mut hasher = Sha256::new();
+    hasher.update(module_name.as_bytes());
+    let result = hasher.finalize();
+
+    Address::from_slice(&result[0..20], prefix)
+}
+
+#[cfg(feature = "ethermint")]
```

Comment:
This replicates the `althea debug addr` function which can be extremely helpful.

Reply:
awesome. I actually have this exact code in a local rust crate I use for tinkering